src/Discovery/DiscoveryDetails/DataDownloadList/ActionButtons/DownloadUtils/DownloadDataFiles/CheckFederatedLoginStatus.tsx

```diff
@@ -17,8 +17,7 @@ interface Iprovider {
 const GUID_PREFIX_PATTERN = /^dg.[a-zA-Z0-9]+\//;
 const CheckFederatedLoginStatus = async (
   setDownloadStatus: (arg0: DownloadStatus) => void,
-  selectedResource: {},
-  manifestFieldName: string,
+  fileManifest: any[],
```

Comment:
The variable name fileManifest confused me a little, it looks like it is an array of fileMetadatas (line 39 below)
Consider renaming to fileMetadataArr or fileMetadataInfo or fileMetadatas

Reply:
I think it would be better if we rename the variable at L39 from `fileMetadata` to `fileManifestItem`
manifest is what we usually call it for a list of file metadata that has GUIDs